eng/Versions.props

```diff
@@ -189,7 +216,7 @@
     <MicrosoftCodeAnalysisCommonPackageVersion>3.7.0-4.20351.7</MicrosoftCodeAnalysisCommonPackageVersion>
     <MicrosoftCodeAnalysisCSharpPackageVersion>3.7.0-4.20351.7</MicrosoftCodeAnalysisCSharpPackageVersion>
     <MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>3.7.0-4.20351.7</MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>
-    <MicrosoftCodeAnalysisFxCopAnalyzersVersion>3.0.0</MicrosoftCodeAnalysisFxCopAnalyzersVersion>
+    <MicrosoftCodeAnalysisFxCopAnalyzersPackageVersion>3.0.0</MicrosoftCodeAnalysisFxCopAnalyzersPackageVersion>
```

Comment:
Change here and at line 255 is about property name consistency. If these properties were automatic, eng/scripts/CodeCheck.ps1 would currently fail; it requires `$(*PackageVersion)` properties for everything in eng/Version.Details.xml. It appears we carried that consistency over to the manual version properties.

If someone (@Pilchie this was your addition) wants to remove `Package` from the names, we can certainly do that -- separately. File an issue and put it on in DoI or Build project. Who knows (@mmitche❔) such a change might even reduce the size of our binlogs or speed builds up a bit 😺